keyboards/czmao/ds17/keyboard.json

```diff
@@ -0,0 +1,93 @@
+{
+    "keyboard_name": "mao_10_DS17_ST",
+    "manufacturer": "CZMAO",
+    "url": "NONE",
+    "maintainer": "MAOKB",
+    "diode_direction": "COL2ROW",
+    "processor": "STM32F103",
+    "bootloader": "stm32duino",
+    "usb": {
+      "vid": "0x4E17",
+      "pid": "0x4E17",
+      "device_version": "1.0.0"
+    },
+    "features": {
+        "bootmagic": true,
+        "command": true,
+        "console": true,
+        "extrakey": true,
+        "mousekey": true,
+        "nkro": true,
+        "rgb_matrix": false,
+        "rgblight": true,
+        "encoder": false
+    },
+    "dynamic_keymap": {
+      "layer_count": 10
+    },
+    
+    "bootmagic": {
+      "matrix" : [0,0]
+    },
+    
+    
+    "matrix_pins": {
+      "cols": ["B12", "B14", "B15", "A8"],
+      "rows": ["A9", "A10", "A15",  "B3",  "B4"]
+    },
+
+    "ws2812": {
+      "pin": "B13"
+    },
+    "rgblight": {
+      "saturation_steps": 8,
+      "brightness_steps": 8,
+      "hue_steps": 10,
+      "led_count": 25,
+      "sleep": true,
+      "animations": {
+          "breathing": true,
+          "rainbow_mood": true,
+          "rainbow_swirl": true,
+          "snake": true,
+          "knight": true,
+          "christmas": true,
+          "static_gradient": true,
+          "rgb_test": true,
+          "alternating": true,
+          "twinkle": true
+      }
+    },
+    "layouts": {
+      "LAYOUT": {
+        "layout": [
```

Comment:
This, and the keymap.c should be reformatted to match the `numpad_5x4` layout, since it matches.

And community layout enabled

Reply:
updated